pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -76,30 +76,44 @@ func SetDefaults_KubeProxyConfiguration(obj *KubeProxyConfiguration) {
 	if obj.ResourceContainer == "" {
 		obj.ResourceContainer = "/kube-proxy"
 	}
-	if obj.IPTablesSyncPeriod.Duration == 0 {
-		obj.IPTablesSyncPeriod = unversioned.Duration{Duration: 30 * time.Second}
+	if obj.IPTablesConfiguration.SyncPeriod.Duration == 0 {
```

Comment:
Maybe follow the data types structure here as well and have one SetDefaults per struct.